src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -197,9 +197,11 @@ enum {
     UCT_IB_MLX5_MD_FLAG_MMO_DMA              = UCS_BIT(15),
     /* Device supports XGVMI UMR workflow */
     UCT_IB_MLX5_MD_FLAG_XGVMI_UMR            = UCS_BIT(16),
+    /* Device supports UAR allocation type (1 for WC, 0 for NC_DEDICATED) */
```

Comment:
with flag name USE_WC no need to explain that 1 is for WC and 0 for NC

Reply:
done